src/document.c

```diff
@@ -3813,3 +3813,14 @@ void document_grab_focus(GeanyDocument *doc)
 
 	gtk_widget_grab_focus(GTK_WIDGET(doc->editor->sci));
 }
+
+static void        *copy_(void *src) { return src; }
+static void         free_(void *doc) { }
+
+/** Gets the GType of GeanyDocument
+ *
+ * @return the GeanyDocument type */
```

Comment:
<rant>
What makes you all think my plugin doesn't want a stable gir interface? It seems to me that there's a notion "let's make it @gironly so we can change it any time", just because these symbols aren't immediately useful for c plugins.

This is not what I wanted. I wanted the gir exposed api to be just a different representation of the original plugin api (baring bugs), with the same stability consideration.

My peasy plugin is depending on the gir in the same way as C API. But you seem to treat it differently, as something that can be broken without consideration.
</rant>

But if that's how it's acceptable for you, so be it. Can add @gironly to this too.